rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -291,18 +339,13 @@ void gs_frame::flip(draw_context_t, bool /*skip_frame*/)
 {
 	static Timer fps_t;
 
-	if (!m_show_fps_in_title && !g_cfg.misc.show_fps_in_title)
```

Comment:
The code below this check is only needed if you have fps enabled or if you just disabled it.
Thats why the check was added. We don't wanna call unnecessary functions all the time.

Reply:
I think it's resolved now. It most should be lightweight, except the actual CallAfter.